### ISSUE TITLE
Version of Ipad??

### ISSUE THREAD

**Issue by bcfager** [petereit/CheckinAir#25]
Thanks so much for your work on this app!! Its so awesome to be able to print direct to printer. I have everything set up for our nursery check-in kiosk. I tried it at my house first and it worked. I connected the printer to the church wireless along with the Ipad 2nd gen ipads ios version 9.1.1 (i think). I log-in to the checkin air app and when I check a child it in doesn't prompt to select a printer. I tried it with my iphone 7 and it works fine. Before buying newer Ipads I want to make sure that is what the issue is. Does the app recognize a device and lock-out other devices on the network? I tried reloading the app, logging in under other users and profiles, even re-set the ipad. All to no avail. I then went home and grabbed my wife's Gen 3 ipad which has IOS 10.3.3 and it still doesn't work. Thoughts??

Bryan

**Comment by petereit** [comment]
Any iPad with iOS 7 or newer should work. Lets make sure it's not an AirPrint issue first.

First, check, double-check and triple-check that your iPhones/iPads and your Brother printer is connected to the same WiFi network. (I fought with this issue once for over an hour before I realized my church had set up a private AND a public WiFi on our router. The printer was connected to the public WiFi but my iPad was on the Private WiFi. LOL!)

Before you even launch the CheckinAir app, open the Safari browser on your iPad 2nd Gen and browse to any site (Google is fine.) Try to print the page. From the print dialog, tap "Select printers"

The AirPrint dialog will scan for available printers. If you can see your Brother label printer, you have it correctly configured.

If it says "No AirPrint Printers Found," you'll know you have a network configuration issue. Either your iPad is not on the right network or your Brother printer is not.

**Comment by petereit** [comment]
I can’t think of any reason it would work on your phone but not your iPad. It’s literally the same code.

**Comment by bcfager** [comment]
Yeah, I am at a loss as well. I tried my wife’s version 3 platform with no luck as well. I am wondering if I will need to order a newer ipad. Like I said previously, they do recognize the printer and worked intermittently but now nothing.

Bryan

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
